templates/page.tpl.php

```diff
@@ -105,10 +105,15 @@
 
 <footer role="contentinfo">
   <div class="wrapper">
-    <section class="footer-top">
-      <?php print render($page['footer_top']); ?>
-    </section>
+    <?php if($page['footer_top']): ?>
+      <section class="footer-top">
+        <?php print render($page['footer_top']); ?>
+      </section>
+    <?php endif; ?>
     <section class="footer-bottom">
+      <div class="footer-logo">
+        <img alt="Australian Government Coat of Arms" src="<?php print drupal_get_path('theme', 'uikit_base'); ?>/images/coat-of-arms.png">
+      </div>
       <?php print render($page['footer_bottom']); ?>
```

Comment:
You should wrap `Australian Government Coat of Arms` in `t()` so it's translatable.  And does `print drupal_get_path(...)` print the opening slash?  You might need to use `base_path() . print drupal_get_path(...)`.